Documentation/CHANGES_IN_V5.md

```diff
@@ -14,9 +14,9 @@ This new container will be dynamically built on the users machine. Users will ne
    `./metroae install vsds -vvv` use `./metroae-container install vsds -vvv`
    Another example for deployments other than defaults
    `./metroae install vsds specialdeployment -vvv` use `./metroae-container install vsds specialdeployment -vvv`
-4. For all image paths, make sure they start with `/metroae`. Here `/metroae` refers to the present working directory for the user.
+4. Users can now specify file locations relative to the MetroAE container using `./` instead of `/metroae`. Here `/metroae` refers to the present working directory for the user. Note that paths to image directories that need to be unzipped prior to MetroAE execution are excluded from this specification.
```

Comment:
I don't think the last part is true. Users should execute ./metroae-container tools unzip images to unzip their image files. When doing this, the nuage-unzip.sh script will be called, which actually requires a relative path